tests/test_regression.py

```diff
@@ -1,100 +1,324 @@
+            os.remove(self.outf_)
+
+class TestPhysorEnrichment(TestRegression):
+    """This class tests the 1_Enrichment_2_Reactor.xml file related to the
+    Cyclus Physor 2014 publication. The number of key facilities, the enrichment
+    values, and the transactions to each reactor are tested.
+    """
+    def __init__(self):
+        super(TestPhysorEnrichment, self).__init__()
+        self.inf = "../input/physor/1_Enrichment_2_Reactor.xml"
+
+    def setup(self):
+        super(TestPhysorEnrichment, self).setup()
+        tbl = self.agent_entry
+        self.rx_id = find_ids(":cycamore:BatchReactor", 
+                              tbl["Spec"], tbl["AgentId"])
+        self.enr_id = find_ids(":cycamore:EnrichmentFacility", 
+                               tbl["Spec"], tbl["AgentId"])
+
+        with tables.open_file(self.outf_, mode="r") as f:
+            self.enrichments = f.get_node("/Enrichments")[:]
         
-        WARNING: the tests require cyclus executable to be included in PATH
-        """
-        for root, dirs, files in os.walk(self.in_dir_):
-            for f in files:
-                if not f.endswith('.xml'):
-                    continue
-                tmp_file = self.tmp_files_[f]
-                run_cyclus("cyclus", os.getcwd(), os.path.join(root, f), 
-                           tmp_file)
-
-                if os.path.isfile(tmp_file):
-                    if f not in sim_files:
-                        continue # nada to do, just making sure it runs
-                    if check_deterministic:
-                        determ = compare_determ(sim_files[f], tmp_file, 
-                                                verbose=True)
-                        assert_true(determ)
-                    else:         
-                        nondeterm = compare_nondeterm(sim_files[f], tmp_file)
-                        assert_true(nondeterm)
-
-                if os.path.isfile(tmp_file):
-                    print("removing {0}".format(tmp_file))
-                    os.remove(tmp_file)
-                
-                tmp_file = tmp_file.split('.')[0] + '.sqlite'
+    def test_deploy(self):
+        assert_equal(len(self.rx_id), 2)
+        assert_equal(len(self.enr_id), 1)
+
+    def test_swu(self):
+        enr = self.enrichments
+        # this can be updated if/when we can call into the cyclus::toolkit's
+        # enrichment module from python
+        exp = [6.9, 10, 4.14, 6.9]
+        obs = [np.sum(enr["SWU"][enr["Time"] == t]) for t in range(4)]
+        np.testing.assert_almost_equal(exp, obs, decimal=2)
+
+    def test_nu(self):
+        enr = self.enrichments
+        # this can be updated if/when we can call into the cyclus::toolkit's
+        # enrichment module from python
+        exp = [13.03, 16.54, 7.83, 13.03]
+        obs = [np.sum(enr["Natural_Uranium"][enr["Time"] == t]) \
+                   for t in range(4)]
+        np.testing.assert_almost_equal(exp, obs, decimal=2)
```

Comment:
It is much more common to do `from numpy.testing import assert_array_almost_equal`, this avoids confusion with `nose.tools.assert_almost_equal`